aiosmtpd/smtp.py

```diff
@@ -195,7 +193,7 @@ def _handle_client(self):
         log.info('%r handling connection', self.session.peer)
         yield from self.push(
             '220 {} {}'.format(self.hostname, self.__ident__))
-        while not self._connection_closed:          # pragma: no branch
+        while self.transport:          # pragma: no branch
```

Comment:
Could you make the check here and below a little more explicit?  Rather than checking truthiness, please use `while self.transport is not None`